src/ga.js

```diff
@@ -175,11 +182,20 @@ function getCpmDistribution(cpm) {
   return distribution;
 }
 
+exports.prefixedSend = function(){
+  if (_trackerName !== null) {
+    return _trackerName + '.send';
+  } else {
+    return 'send';
+  }
+}
+
+
 function sendBidRequestToGa(bid) {
   if (bid && bid.bidderCode) {
     _analyticsQueue.push(function () {
       _eventCount++;
-      window[_gaGlobal]('send', 'event', _category, 'Requests', bid.bidderCode, 1, _disibleInteraction);
+      window[_gaGlobal]( prefixedSend, 'event', _category, 'Requests', bid.bidderCode, 1, _disibleInteraction);
```

Comment:
How about `trackerSend` which describes tracker name and method used. "Prefixed" describes what happened with the code but not what the code is doing, imo.